docs/guide/ingress/annotations.md

```diff
@@ -446,7 +446,7 @@ ALB supports authentication with Cognito or OIDC. See [Authenticate Users Using
           namespace: testcase
           name: my-k8s-secret
         data:
-          clientID: base64 of your plain text clientId
```

Comment:
This change was deliberate in the AWS Load Balancer controller. We still support clientId for backwards compatibility with the AWS ALB Ingress controller.

Reply:
Thanks, closing!